docker-compose/config.local.env

```diff
@@ -61,4 +61,4 @@ MONGOCK_MIGRATIONSCANPACKAGE_0=be.vlaanderen.informatievlaanderen.ldes.server.in
 MONGOCK_MIGRATIONSCANPACKAGE_1=be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset2
 MONGOCK_MIGRATIONSCANPACKAGE_2=be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset3
 MONGOCK_MIGRATIONSCANPACKAGE_4=be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset5
-MONGOCK_MIGRATIONSCANPACKAGE_5=be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset6
```

Comment:
why are you changing this?

Reply:
mistake .. although we shouldn't be keeping this. we don't need this list.